sql/hive/src/test/scala/org/apache/spark/sql/sources/BucketedWriteSuite.scala

```diff
@@ -65,6 +66,11 @@ class BucketedWriteSuite extends QueryTest with SQLTestUtils with TestHiveSingle
 
   private val df = (0 until 50).map(i => (i % 5, i % 13, i.toString)).toDF("i", "j", "k")
 
+  def tableDir: File = {
```

Comment:
This looks reasonable, just curious about why the previous one not working after your PR....